assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -237,27 +233,29 @@ export default function UserInputQuestionnaire() {
 		<div>
 			{ settingsProgress }
 
-			{ activeSlugIndex <= steps.indexOf( USER_INPUT_QUESTION_ROLE ) && (
+			{ activeSlugIndex <=
+				steps.indexOf( USER_INPUT_QUESTIONS_PURPOSE ) && (
 				<UserInputQuestionWrapper
-					slug={ USER_INPUT_QUESTION_ROLE }
-					isActive={ activeSlug === USER_INPUT_QUESTION_ROLE }
+					slug={ USER_INPUT_QUESTIONS_PURPOSE }
+					isActive={ activeSlug === USER_INPUT_QUESTIONS_PURPOSE }
 					questionNumber={ 1 }
 					title={ __(
-						'Which best describes your team/role in relation to this site?',
+						'What is the main purpose of this site?',
 						'google-site-kit'
 					) }
 					description={ __(
-						'This will help Site Kit show tips that help you specifically in your role',
+						'Based on your answer, Site Kit will tailor the metrics you see on your dashboard to help you track how close you’re getting to your specific goals',
 						'google-site-kit'
 					) }
 					next={ nextCallback }
 					nextLabel={ nextLabel }
+					back={ backCallback }
```

Comment:
Is there a reason this `back` prop is added to the first question? The `main` branch and the Figma designs do not have a `Back` action for the first question, and clicking on it doesn't do anything.

Reply:
Hi @nfmohit . 
This was copy-paste from the original source with `back={ backCallback }` already. I guess it acts inside `settings screen`.

Reply:
Thank you @sashadoes. I believe we need to remove it for the first question because it never existed there until now, and doesn't exist in the final designs as well.

Since this is such a small change, I've added [a small commit](https://github.com/google/site-kit-wp/pull/6062/commits/889f97978b9892577b1d716370d52a8bdb4a83c3) to quickly address it and reduce the back-and-forth.

Reply:
@nfmohit got it! Thank you 🙂